Automattic/jetpack
# Podcast Player Block: Add Pocket Casts variation.

#### Changes proposed in this Pull Request:

* This experimental PR adds Pocket Casts variation to the Podcast Player Block which includes branding & link to Pocket Casts.

| Jetpack  | Pocket Casts |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/1287077/168567992-da95c9fd-9ec1-4de3-90d2-9538a4d0d3f3.png) | ![image](https://user-images.githubusercontent.com/1287077/168568006-fd1c2554-fbd8-46e6-8e42-c05f18fc1ed5.png) |
| ![image](https://user-images.githubusercontent.com/1287077/168568099-793e055a-b5ea-4940-a497-cc562b558100.png) | ![image](https://user-images.githubusercontent.com/1287077/168568128-ad528cb2-24b3-42af-98da-4016e04f7179.png) |
| ![image](https://user-images.githubusercontent.com/1287077/168570994-3e88a7a8-b0f1-49de-ab20-652da2338c53.png) |

#### Jetpack product discussion


To be updated.

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






_This can't be tested locally. Jurassic Ninja didn't work for me either on the URL to RSS resolving part._

* Apply a phab patch on sandbox.
* Add "Pocket Casts Player" block.
* Paste a Pocket Casts show link, e.g. `https://pca.st/stdwytkaudio`
* The Pocket Casts Player block should render with:
  * PocketCasts Logo on footer.
  * **Open In Pocket Casts** button on the footer.
  * Red & Black color scheme.
* Clicking on **Open In Pocket Casts** logo should go to the pocket casts site in another tab.

#### TO-DO:

Design:
- [ ] Should we still proceed with this idea or just rely solely on oEmbed?
- [ ] Check with designers on how they want this Pocket Casts variation of the player block to look.

Technical:
- [ ] Check if modifications are needed on the dynamic block `render_callback`.
- [ ] Prevent non-pocket-casts link from being pastable on the Pocket Casts variation
- [ ] Pocketcasts link pasted on default podcast player block should change to Pocket Casts variation.
